addons/editor/initKeybinds.sqf

```diff
@@ -100,3 +100,14 @@
         };
     };
 }, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+// Hack fix for the Zeus lockup bug, will reopen Zeus interface.
+[ELSTRING(main,DisplayName), QGVAR(reloadDisplay), [LSTRING(ReloadDisplay), LSTRING(ReloadDisplay_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+        findDisplay 312 closeDisplay 2;
+
+        {openCuratorInterface} call CBA_fnc_execNextFrame;
```

Comment:
```suggestion
        [{ openCuratorInterface }] call CBA_fnc_execNextFrame;
```

Reply:
There is already another spot where it is done like this and works: https://github.com/zen-mod/ZEN/blob/e2a7f8b695a4d19a4880dc1db6be9a10b43f294c/addons/remote_control/functions/fnc_start.sqf#L79

Reply:
I think that @mharis001 has a way of always putting things in `[]` even if there is only one argument.

Reply:
Well, that is his code according to blame.